.ci/azure/linux_onnxruntime.yml

```diff
@@ -67,6 +67,7 @@ jobs:
       python3 -m pip install -r $(REPO_DIR)/inference-engine/ie_bridges/python/requirements.txt
       # For running Python API tests
       python3 -m pip install -r $(REPO_DIR)/inference-engine/ie_bridges/python/src/requirements-dev.txt
+      python3 -m pip install cmake>=3.18
```

Comment:
why do we need non-default cmake?

Reply:
that was required for onnxruntime build. But it's not anymore since the onnxruntime branch was fixed to 1.8.1